### ISSUE TITLE
v2.64.4 - Error with Internet Explorer 11

### ISSUE THREAD

**Issue by jbrule** [kaltura/mwEmbed#3685]
The following error is presented when loading the player v2.64.4 in Internet Explorer 11 (Document Mode 'Edge'). This error was not present with the previous version we ran (2.63.3).

`SCRIPT438: Object doesn't support property or method 'hideTextTrack'`

**Comment by kaltura-hooks** [comment]
Hi @jbrule,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa \"kaltura*\"
```	
	For deb based systems:
```	
	# dpkg -l \"kaltura-*\"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color \"ERR:\|PHP\|trace\|CRIT\|\[error\]\"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by itaykinnrot** [comment]
Thanks @jbrule - we're checking

**Comment by itaykinnrot** [comment]
Fixed in 2.66 branch
